electron-builder.json5

```diff
@@ -411,6 +411,7 @@
       Comment: 'Delta Chat email-based messenger',
       Keywords: 'dc;chat;delta;messaging;messenger;email',
     },
+    description: 'The Email messenger\nWith Delta Chat, you can use your existing Email account as a messenger. End-to-End Encryption with Autocrypt is supported and you can have interactive chat experiences using webxdc apps(https://webxdc.org).',
```

Comment:
nope this is not right place for it, look at the first line, this file is auto generated